src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/PathUtilities.cs

```diff
@@ -56,7 +56,6 @@ internal static int IndexOfFileName(string path)
         /// <summary>
         /// Get file name from path.
         /// </summary>
-        /// <remarks>Unlike <see cref="System.IO.Path.GetFileName"/> doesn't check for invalid path characters.</remarks>
```

Comment:
Why is this comment being deleted?

Reply:
It leads to Ambiguous reference in cref attribute as there are two overloads. I have changed it to System.IO.Path.GetFileName(string)